src/components/Auth/Buttons/SignUpButton.js

```diff
@@ -0,0 +1,23 @@
+import { Button } from '@mui/material';
+import { useAuth0 } from '@auth0/auth0-react';
+import React from 'react';
+
+const SignUpButton = () => {
+  const { loginWithRedirect } = useAuth0();
+
+  const handleSignUp = async () => {
+    await loginWithRedirect({
+      authorizationParams: {
+        screen_hint: 'signup',
+      },
+    });
+  };
+
+  return (
```

Comment:
The three buttons here Login, Logout, and Signup should probably be combined in some way maybe login and signup in one button and logout separate or just a single button that is reused and uses props to determine its text and functionality.